src/services/refactors/convertArrowFunctionOrFunctionExpression.ts

```diff
@@ -141,7 +141,7 @@ namespace ts.refactor.convertArrowFunctionOrFunctionExpression {
         const token = getTokenAtPosition(file, startPosition);
         const typeChecker = program.getTypeChecker();
         const func = tryGetFunctionFromVariableDeclaration(file, typeChecker, token.parent);
-        if (func && !containingThis(func.body)) {
+        if (func && !containingThis(func.body) && !typeChecker.containsArgumentsReference(func)) {

```

Comment:
Do these walk the function twice? Probably not a huge deal right now, but it would be smarter to use our find-all-references logic long-term.

Reply:
Oke. Will check it later.